src/cgr_gwas_qc/workflow/scripts/concordance_table.py

```diff
@@ -58,16 +69,65 @@ def read(filename: PathLike) -> pd.DataFrame:
     return pd.read_csv(filename, dtype=DTYPES)
 
 
-def build(filename: Path, concordance_threshold: float, pi_hat_threshold: float) -> pd.DataFrame:
+def _sort_ids(x: pd.DataFrame):
+    """ "Sort IDs alphanumerically."""
+    x.IID1, x.IID2 = np.where(x.IID1 < x.IID2, [x.IID1, x.IID2], [x.IID2, x.IID1])
+    x.rename(columns={"IID1": "ID1", "IID2": "ID2"}, inplace=True)
+    return x
+
+
+def prep_concordance_table(pi_hat_threshold: float, concordance_threshold: float, x: pd.DataFrame):
+    """ "Prepares the concordance_table by appending new columns based on thresholds."""
     return (
-        plink.read_genome(filename)
+        _sort_ids(x)
         .assign(is_ge_pi_hat=lambda x: x.PI_HAT >= pi_hat_threshold)
         .assign(concordance=lambda x: x.IBS2 / (x.IBS0 + x.IBS1 + x.IBS2))
         .assign(is_ge_concordance=lambda x: x.concordance >= concordance_threshold)
         .reindex(DTYPES.keys(), axis=1)
     )
 
 
+def process_genome_chunk(pi_hat_threshold: float, concordance_threshold: float, chunk):
+    """ "Processes each chunk of genome file."""
```

Comment:
Remove extra the extra `"`.

Reply:
Removed in 9b57d4395aaaf5eabda8ad3b1d218dc76a358490